models/demos/t3000/mixtral8x7b/tt/mixtral_common.py

```diff
@@ -83,41 +83,31 @@ def prepare_inputs_ttnn(x_bsh, hidden_size, current_pos, sliding_window, device_
     )
 
     # Attention mask
-    padded_layer_past_len = min(nearest_32(current_pos + 1), sliding_window)
-    current = current_pos % sliding_window
-    n_local_heads = 4  # model_args.n_heads // 8 # num_devices TODO pass the arguments instead of hard coding
-
+    padded_layer_past_len = nearest_32(current_pos + 1)
     attn_mask = torch.zeros(seq_len, 32, 32, padded_layer_past_len)  # [SB4P]
```

Comment:
The attn_mask does not have this shape, right? [SB4P]

Also, for the batch, instead of hardcoding, can we pass the value from somewhere else?